utils/demo/helpers/addManipulationBindings.ts

```diff
@@ -0,0 +1,70 @@
+import * as cornerstoneTools from '@cornerstonejs/tools';
```

Comment:
I guess you need to git pull origin

Reply:
?? Not sure what you mean here.  The import is done in other helpers files, so I don't see the issue with this one either?